filetypes/servicenow.json

```diff
@@ -0,0 +1,60 @@
+{
+  "filetype": {
+    "sn_hr_core_case": {
+      "icon": "case",
+      "captions": {
+        "en": "HR Case",
+        "fr": "Requête RH"
+      }
+    },
+    "sc_cat_item": {
+      "icon": "custom",
+      "captions": {
+        "en": "Catalog Item",
+        "fr": "Item du Catalogue"
+      }
+    },
+    "sn_customerservice_case": {
+      "icon": "case",
+      "captions": {
+        "en": "Case",
+        "fr": "Requête"
+      }
+    },
+    "kb_social_qa_answer": {
+      "icon": "list",
```

Comment:
FYI, we have a ton of available images, including all those available from the salesforce design system you can choose from.

I'm pretty certain there's a dedicated/better icon for most of these instead of generic ones.

Reply:
I checked the one we have that aren't Salesforce and there isn't much. I'm not sure I want to show Salesforce Images within ServiceNow.